index.html

```diff
@@ -1,76 +1,223 @@
+        </p>
+        <p>
+          <a href="/sdks/cordova/" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/sdks/ios/"><i class="fa fa-apple"></i> iOS SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/sdks/ios/" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/sdks/xamarin/"><i class="font-icon icon-xamarin"></i> Xamarin SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/sdks/xamarin/" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="">React Native SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
         <p>
-          <a href="http://www.feedhenry.com/feedhenry-and-red-hat-pushing-ahead-with-integrations/"><img src="{{ site.baseurl }}/img/logos_home/feedhenry.gif" alt="feedhenry"  /></a>
-          <a href="https://developers.openshift.com/en/xpaas-unified-push.html"><img src="{{ site.baseurl }}/img/logos_home/openshift.gif" alt="openshift"  /></a>
+          <a target="_blank" href="https://github.com/feedhenry/fh-js-sdk" class="btn btn-primary">Learn More</a>
         </p>
-      </div><!-- container -->
-    </section>
\ No newline at end of file
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="">PWA SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a target="_blank" href="https://github.com/feedhenry/fh-js-sdk" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="custommers">
+  <div class="container">
+    <h2 class="section-header">Mobile development begins here.</h2>
+    <p>Using mobile technologies will never be so easy as in the cloud. AeroGear supports several cloud-enabled providers and
```

Comment:
This sentence here seems to be out of date. Do we have something to update this with @jasonmadigan @laurafitzgerald? I think it would look nicer centered 
![image](https://user-images.githubusercontent.com/9078522/41361818-dc742d38-6f27-11e8-84e4-10a1ecdf754c.png)

Reply:
Reuse the whole paragraph?

Reply:
@dimitraz yeah, lets use the whole paragraph

Reply:
Ok